crates/check/tests/solution.rs

```diff
@@ -391,7 +391,7 @@ async fn predicate_graph_state_read() {
             // Push the length, num keys to read and slot index for the `KeyRange` op.
             PUSH(4),
             PUSH(1),
-            PUSH(0),
+            // PUSH(0),
```

Comment:
?

Reply:
Ah this was no longer necessary as the following `ALOC` adds the mem_addr onto the stack for us - removed this and improved the comments.